lldb/tools/lldb-dap/Handler/StackTraceRequestHandler.cpp

```diff
@@ -180,44 +184,53 @@ void StackTraceRequestHandler::operator()(
   llvm::json::Object body;
 
   lldb::SBFormat frame_format = dap.frame_format;
+  bool include_all = false;
```

Comment:
Should this default to `dap.configuration.displayExtendedBacktrace` and then it simplifies line 75-77.